README.md

```diff
@@ -120,6 +120,84 @@ make run
 make test
 ```
 
+## Local stack setup
+
+For running the Court data adaptor and a Mock common platform API locally
+
+- generate OAuth2 `client_credentials` for the UI, via the adaptor
+```
+cd .../laa-court-data-adaptor
+git checkout master
+git pull
```

Comment:
minor: master is the default branch and the repo would have have been cloned, is this required?